atomic_reactor/inner.py

```diff
@@ -668,12 +659,22 @@ def build_process_failed(self):
 
     def throw_canceled_build_exception(self, *args, **kwargs):
         self.data.build_canceled = True
+        from atomic_reactor.plugin import BuildCanceledException
         raise BuildCanceledException("Build was canceled")
 
     def build_docker_image(self) -> None:
         """
         build docker image
         """
+        from atomic_reactor.plugin import (
+            BuildStepPluginsRunner,
+            ExitPluginsRunner,
+            PluginFailedException,
+            PostBuildPluginsRunner,
+            PreBuildPluginsRunner,
+            PrePublishPluginsRunner,
+        )
```

Comment:
Is this the only way to get around the cyclic import? It feels strange to do it like this :(

Reply:
PTAL